### PR TITLE
Added 2 additional servers

### PR DESCRIPTION
**Pull request by dansyuh**
Added Fyre Network & MineKings Network

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -788,5 +788,22 @@
         "addresses": [
             "originrealms.com"
         ]
+    },
+    {
+        "id": "fyre",
+        "name": "Fyre",
+        "addresses": [
+            "fyre.cc",
+            "play.fyre.cc"
```
Comment (GrowlyX) [inline servers.json:797]: Please take a look at:
[the ip-address section in the README](https://github.com/LunarClient/ServerMappings/blob/master/README.md#ip-addresses)

You do not need to add sub-domains to the addresses array.

File: servers.json [hunk 2]
```diff
@@ -788,5 +788,22 @@
...
+        ]
+    },
+    {
+        "id": "minekings",
+        "name": "MineKings",
+        "addresses": [
+            "minekings.org",
+            "play.minekings.org",
```
Comment (GrowlyX) [inline servers.json:805]: Please remove these as well

**Review by GrowlyX — CHANGES_REQUESTED** [review]
Remove the un-needed subdomains in the `fyre` and `minekings` entries

**Comment by DanielCrewe** [comment]
Hey, considering I removed the GitHub repository from my GitHub account I have proposed another pull request, thanks.

**Pull request closed without merging**